modules/data.land/R/dataone_download.R

```diff
@@ -27,24 +27,28 @@ dataone_download = function(id, filepath = "/fs/data1/pecan.data/dbfiles/", CNod
   ### automatically retrieve mnId
   cn <- dataone::CNode(CNode) 
   locations <- dataone::resolve(cn, pid = id) 
+    print("Connecting to Member Node")
```

Comment:
Use the PEcAn logger package, rather than print, to post messages as this will go to the terminal or correct log file at the correct level of verbosity (e.g. these mostly look like logger.info, not warnings or errors)

Reply:
Got it. Just pushed these changes.